pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -2,7 +2,9 @@
 
 .wrapper
   = view :unsupported_browser
-  -if header['theme'] == 'responsive'
+  -if request.cookies['pm'] == 'new_header'
```

Comment:
This is `pagemode`, an existing cookie with special cache treatment (used for earlier Hour of Code I believe).